test/api/unit/services/GitHub.test.js

```diff
@@ -676,4 +676,77 @@ describe('GitHub', () => {
       expect(result).to.be.undefined;
     });
   });
+
+  describe('.getContent', () => {
+    /* eslint-disable camelcase */
+    it('returns a file based on the supplied parameters', async () => {
+      const accessToken = 'token';
```

Comment:
for the values that don't change from test to test, you can just define them once in the `describe` block...